pyensign/ensign.py

```diff
@@ -248,6 +248,12 @@ async def subscribe(self, *topics, query="", consumer_group=None):
         ):
             yield event
 
+    async def query(self, query, params):
```

Comment:
Unimplemented for now, but we should create a story to make these work.

Reply:
@BeciLambrecht can you please help make sure that story gets created?